app/src/main/java/com/bogdan/codeforceswatcher/LoadUser.kt

```diff
@@ -0,0 +1,89 @@
+package com.bogdan.codeforceswatcher
+
+import android.support.v4.widget.SwipeRefreshLayout
+import android.util.Log
+import com.bogdan.codeforceswatcher.activity.MainActivity
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import java.util.concurrent.CountDownLatch
+
+object LoadUser {
+
+    var textNotification = ""
+
+    fun loadUser(handle: String, swipeRefresh: SwipeRefreshLayout?) {
```

Comment:
Passing `swipeRefresh` view to this method is not a good idea, because it works only for UI and actually isn't very related to logic. To do it better you can return `countDownLatch` from this method and then use `await` to hide `swipeRefresh` in UI.

Reply:
Or the best option is to pass some `callback` into this method and call it when refreshing is done.